src/styles/untiles/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$white-color:#fff;
+$arial-color:#00acdc;
+$bold-weight:700;
+$bludark-color:#060b35;
+$width-section: 166px;
```

Comment:
The variable name 'width-section' might not be descriptive enough. Consider renaming it to 'card-width' or another name that better describes its purpose.